configureBuild.sh

```diff
@@ -255,6 +259,7 @@ processArgumentsforSpecificArchitectures() {
     fi
     if [[ ${BUILD_CONFIG[USER_SUPPLIED_CONFIGURE_ARGS]:-""} != *"--with-jobs"* ]]; then
       BUILD_CONFIG[USER_SUPPLIED_CONFIGURE_ARGS]="--with-jobs=${BUILD_CONFIG[NUM_PROCESSORS]} ${BUILD_CONFIG[USER_SUPPLIED_CONFIGURE_ARGS]:-''}"
+      BUILD_CONFIG[USER_SUPPLIED_CONFIGURE_ARGS]="--with-jobs=8 ${BUILD_CONFIG[USER_SUPPLIED_CONFIGURE_ARGS]:-''}"
```

Comment:
why 8?

Reply:
I think that was in there because I'd hard coded for the machine I was testing on for maximum throughput. Removed.